features/cli_version.feature

```diff
@@ -4,7 +4,7 @@ Feature: Howitzer CLI Version
     When I run `howitzer <option>`
     Then the output should contain exactly:
     """
-    howitzer version <HOWITZER_VERSION>
+    howitzer version 2.0.3
```

Comment:
The test will be broken next release. Please convert transformer to new format in Cucumber acceptance tests also

Reply:
same